File: Container_loop/4.py
input_id=input("아이디를 입력해주세요. \n")
ids = ['har', 'space', 'SDM']
for id in ids:
    if id == input_id:
        print('Hello!, ' +id)
        import sys
        sys.exit()

print('who are you?')

File: Container_loop/4.rb
puts("아이디를 입력해주세요.")
input_id = gets.chomp()
ids = ['har', 'space', 'SDM']
for id in ids do
    if id == input_id
        puts('Hello!, ' +id)
        exit
    end
end
puts('who are you?')
